taskipy/run.py

```diff
@@ -20,36 +20,38 @@ def run_commands_and_bail_on_first_fail(cmds: List[str]) -> int:
     except FileNotFoundError:
         print('no pyproject.toml file found in this directory')
         sys.exit(1)
-    except:
+    except Exception:  # pylint: disable=W0703
```

Comment:
If we follow the the pattern we've set with `except KeyError`, how about we intercept `toml.TomlDecodeError` here? wdyt?

Reply:
I didn’t know which exception to catch!